netket/models/rbm.py

```diff
@@ -240,6 +241,8 @@ def __call__(self, x_in):
             precision=self.precision,
         )(x_in)
         x = self.activation(x)
+
+        x = x.reshape(-1, self.features * self.n_symm)
```

Comment:
why is the reshape needed?

Reply:
This combines the feature and symmetry dimensions, which was original done inside of DenseSymm.

Reply:
i noticed later.
I think it's ok. Though i'd like to check with @femtobit .

BTW, this is a breaking change. 
It actually does break a bunch of my code (because i use DenseSymm directly).
Usually if you want to make such a change you should highlight it in the PR description. 
If then we miss it, it might create a mess for the users

Reply:
Definitely, I'll be sure to be more careful in the future.